doc/sphinx/locale/es/LC_MESSAGES/external_incoming_calls.po

```diff
@@ -664,9 +664,9 @@ msgid ""
 "messages to destination **SIP Proxy**, they will be sent to the IP:PORT "
 "of this field."
 msgstr "Normalmente se deja vacío o se pone la IP del dominio indicado en"
-"**SIP Proxy** (para evitar resolución de dominios y hacer que el mensaje"
+"**SIP Proxy** (para evitar resolución de dominios y hacer que el mensaje "
```

Comment:
An extra space at the beginning of this line or at the end of the previous one will make the bold SIP Proxy work as expected.